addons/dragging/functions/fnc_canCarry.sqf

```diff
@@ -22,24 +22,24 @@ if !(alive _target && {_target getVariable [QGVAR(canCarry), false]} && {isNull
 
 if !([_unit, _target, []] call EFUNC(common,canInteractWith)) exitWith {false};
 
-//#2644 - Units with injured legs cannot bear the extra weight of carrying an object
-//The fireman carry animation does not slow down for injured legs, so you could carry and run
+// #2644 - Units with injured legs cannot bear the extra weight of carrying an object
+// The fireman carry animation does not slow down for injured legs, so you could carry and run
 if ((_unit getHitPointDamage "HitLegs") >= 0.5) exitWith {false};
 
 // Static weapons need to be empty for carrying (ignore UAV AI)
 if (_target isKindOf "StaticWeapon") exitWith {
-    crew _target findIf {getText (configOf _x >> "simulation") != "UAVPilot"} == -1
+    (crew _target) findIf {getText (configOf _x >> "simulation") != "UAVPilot"} == -1
 };
 
 // Units need to be unconscious or limping
 if (_target isKindOf "CAManBase") exitWith {
-    lifeState _target isEqualTo "INCAPACITATED"
-    || {_target getHitPointDamage "HitLegs" >= 0.5}
+    lifeState _target == "INCAPACITATED" ||
```

Comment:
Pointless and [lowers performance](https://community.bistudio.com/wiki/isEqualTo).

Reply:
> Pointless and [lowers performance](https://community.bistudio.com/wiki/isEqualTo).

It's for readability reasons. I'm well aware it performs faster, but I feel in these sorts of cases the readability outweighs the performance loss, especially since there is virtually none here.